src/modules/storage.js

```diff
@@ -57,3 +65,56 @@ window.addEventListener('DOMContentLoaded', () => {
   }
   addList();
 });
+
+// input field of list clicked event
+
+/*
+listsList.addEventListener('focusin', (e) => {
+  if(e.target.className.includes('single-list-input')){
+    const parent = e.target.parentNode.parentNode;
+    parent.querySelector('.single-list-action-button').innerHTML =
+    '<button class = "delete-btn" onclick="removeList('${list}')"> // eslint-disable-line
+    <i class="fa-solid fa-trash-can"></i></button>';
+  }
+})
+listsList.addEventListener('focusout', (e) => {
+  if(e.target.className.includes('single-list-input')){
+    const parent = e.target.parentNode.parentNode;
+    parent.querySelector('.single-list-action-button').innerHTML = '<button class = "move-btn">
+    <i class="fa-solid fa-ellipsis-vertical"></i></button>';
+
+    const inputValue = e.target.value
+    const {id} = parent.id;
+
+    listArray.forEach(list => {
+      if(id === list.index){
+        list.description = inputValue
+      }
+    })
+
+    localStorage.setItem('listData', JSON.stringify(listArray));
+  }
+})
```

Comment:
- [ ] Please kindly make sure there is no commented code in your codebase as it makes the codebase unclean.